Commands/InteractionServiceHandler.cs

```diff
@@ -25,7 +26,7 @@ public static async Task InitAsync()
 		}
 		catch
 		{
-			await Program.LogAsync("InteractionServiceHandler", "Error initializing InteractionService", LogSeverity.Critical);
+			Log.Fatal("Error initializing InteractionService");
```

Comment:
Provide additional information about the error

```suggestion
			Log.Fatal($"Error initializing InteractionService: {e.Message}");
```